test/jtreg-ext/requires/VMProps.java

```diff
@@ -513,7 +513,11 @@ protected String dockerSupport() {
         return "" + isSupported;
     }
 
-    private void redirectOutputToLogFile(String msg, ProcessBuilder pb, String fileNameBase) {
+    // Returns comma-separated file names for stdout and stderr.
+    private String redirectOutputToLogFile(String msg, ProcessBuilder pb, String fileNameBase) {
+        if (!Boolean.getBoolean("jtreg.log.vmprops")) {
+            return "";
```

Comment:
I think having to explicitly enable this somewhat defeats the purpose of using it for diagnostics. If we get an unexpected failure we will have to reproduce it with logging enabled to see what went wrong.

Reply:
If I remove this conditional guard then we will see extra output at the beginning of test run which might be undesirable. For instance: 
----------------
jtreg -v1 -conc:4 -Djtreg.log.vmprops=true open/test/hotspot/jtreg/containers/docker/                    

VMProps: Entering call()
VMProps: Entering dockerSupport()
VMProps: dockerSupport(): platform check: isSupported = false
VMProps: dockerSupport(): returning isSupported = false
VMProps: Leaving call()
Test results: no tests selected
-----------------
I am not sure what is the best thing to do here, so I stayed on a "safe" side and added this conditional guard. The model in my mind is that such logging is normally off but can be turned on in case the issue is suspected. However if you believe it should be always on I can remove the guard.

Reply:
The failure happens so rarely I think you need to leave the option on in order to get
any useful information about this issue when it occurs in Mach5. If a sighting happens
in Mach5, I don't think rerunning another task on the same machine will necessarily
fail so you need to get the information from the failures that do occur.

Reply:
Thanks Dan and David. I will remove the condition then, and have this logging always on.